.gitbook.yaml

```diff
@@ -0,0 +1,5 @@
+root: ./docs/
+
+​structure:
+  readme: README.md
+  summary: SUMMARY.md​
```

Comment:
What's the strange parsing here indicating? Incorrect yaml?

Reply:
Speediest review!! My copy and paste from another file might have gone weird. Just pushed a fix.